.github/workflows/west-commands.yml

```diff
@@ -49,6 +49,8 @@ jobs:
       - name: Install requirements
         shell: bash
         run: |
+          #pip install --upgrade pip
```

Comment:
comment can be removed

Reply:
Unaddressed

Reply:
but it's minor thing, if no more changes needed, we can merge with it

Reply:
addressed